### PR TITLE
chore: experiments

### PR DESCRIPTION
**Pull request by josepot**
let's start with this and we can keep improving them... is that cool?

### CODE REVIEW

**Review by kratico — APPROVED** [review]
LGTM

**Comment by kratico** [comment]
Perhaps, we could have a `experiments/` as a sibling of `packages/`

Thoughts?

**Comment by josepot** [comment]
> Perhaps, we could have a `experiments/` as a sibling of `packages/`
> 
> Thoughts?

yeah! I like that better, for sure!

File: packages/experiments/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "experiments",
```
[the remark this answers is not in the document]
Comment (josepot) [reply packages/experiments/package.json:2]: good point!

File: package.json [hunk 2]
```diff
@@ -2,7 +2,8 @@
   "private": true,
   "license": "MIT",
   "workspaces": [
-    "packages/*"
+    "packages/*",
+    "experiments"
```
Comment (kratico) [inline package.json:6]: `experiments/*` will enable parallel experiments 😄

**Review by kratico — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (josepot) [reply package.json:6]: ahhhhhh... of course! right! I mean is it really worth it? isn't it better to have them all under the same "project"? 🤔 I'm not sure, though... Anyways, go for it if you want, for sure! Run the experiment! 🙈

**Pull request merged**